slices/rabbitmq-server.yaml

```diff
@@ -0,0 +1,114 @@
+package: rabbitmq-server
+
+essential:
+  - rabbitmq-server_copyright
+
+slices:
+  autocomplete:
+    contents:
+      /usr/lib/rabbitmq/autocomplete/*:
+
+  bins:
+    essential:
+      - base-files_base
+      - base-passwd_data
+      - bash_bins
+      - coreutils_bins
+      ## erlang* packages do not exist for i386
+      ## for all other archs, uncomment all those slices below
+      # - erlang-base_bins
+      # - erlang-inets_bins
+      # - erlang-os-mon_bins
+      - grep_bins
+      - libc-bin_locale
+      - libc-bin_nsswitch
+      - libtinfo6_libs
+      - login_bins
+      - netbase_default-hosts
+      - netbase_default-networks
+      - passwd_bins
+      - python3.12_standard
+      - rabbitmq-server_libs
+      - sed_bins
+      - util-linux_bins
+    contents:
```

Comment:
do you need all of these? and again, some sound like scripts and not binaries

Reply:
Yes, most of those are bash scripts or just hardlinkes, but yeah they are needed. I tried for instance removing the rabbitmq-plugins script to create a smaller image out of that slice, but that broke rabbitmq-server / rabbitmqctl.